app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
+
+  def increment_post_likes_counter
+    post.increment!(:like_counter)
+  end
```

Comment:
[OPTIONAL] And also here, you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* after_save to call the `#increment_post_likes_counter`